src/blocks/share/index.js

```diff
@@ -15,19 +14,15 @@ const { __, _x } = wp.i18n;
 /**
  * Block constants
  */
-const { name } = metadata;
+const name = 'social';
 
 const title = __( 'Share' );
 
 const icon = icons.social;
 
-const keywords = [
-	__( 'social' ),
-	__( 'coblocks' ),
-];
+const keywords = [ __( 'social' ), __( 'coblocks' ) ];
```

Comment:
@richtabor We should rethink the keywords here. We are limited to just three but we don't need to add 'social' because it's the block name, and we don't need to add 'coblocks' because it's within that group. Both of these string locations are handled by search.

Reply:
This is now `[ __( 'share' ), __( 'links' ), __( 'icons' ) ];`